cmds/core/uptime/uptime.go

```diff
@@ -11,26 +11,32 @@ package main
 import (
 	"errors"
 	"fmt"
+	"io"
 	"log"
 	"os"
 	"strings"
 	"time"
 )
 
+var (
+	errAverageFormat = errors.New("the contents of proc/loadavg we are trying to process contain less than the required 3 loadavgs")
+	errUptimeFormat  = errors.New("the contents of proc/uptime we are trying to read are empty")
+)
+
 // loadavg takes in the contents of proc/loadavg,it then extracts and returns the three load averages as a string
 func loadavg(contents string) (loadaverage string, err error) {
 	loadavg := strings.Fields(contents)
 	if len(loadavg) < 3 {
-		return "", fmt.Errorf("error:invalid contents:the contents of proc/loadavg we are trying to process contain less than the required 3 loadavgs")
+		return "", errAverageFormat
```

Comment:
a log of times, in logfiles, it's nice to see the data we think is defective
return "", fmt.Errorf("%q:%v", contents, errAverageFormat)